src/components/InfoTooltip.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+	<span class="querybuilder__Tooltip">
+		<Popover
+			:reactToHover="true"
+			:position="position"
+		>
+			<template v-slot:target>
+				<Button
+					variant="quiet"
+					type="neutral"
+					iconOnly
+					aria-label="Tooltip">
+					<Icon type="info-outlined" size="small" />
+				</Button>
+			</template>
+			<div v-html="message"/>
+		</Popover>
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Icon, Button, Popover } from '@wmde/wikit-vue-components';
+
+export default Vue.extend( {
+	name: 'InfoTooltip',
+	components: {
+		Icon,
+		Button,
+		Popover,
+	},
+	props: {
+		position: {
+			type: String,
+			required: true,
+		},
+		message: {
+			type: String,
+			required: true,
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+.querybuilder__Tooltip {
+	& .wikit-Popover__target .wikit.wikit-Button--iconOnly.wikit-Button--neutral {
+		padding-block: 0;
+		padding-inline-start: 0;
+		line-height: 1;
+
+		&:focus {
```

Comment:
There is still a jumpy thing with a border that looks strange. If you click on the icon and then outside you can briefly see a jump and a black border. 

I tried with:
``` 
& .wikit-Popover__target .wikit.wikit-Button--iconOnly.wikit-Button--neutral {
		padding-block: 0;
		padding-inline-start: 0;
		line-height: 1;
		border: 0;

		&:focus {
			box-shadow: none;
		}

	}
```

and it works. 

I am not sure about the margin still, because there is a gray area around the icon and you can still see it is aligned to the start. I can see it in the css above, why did you decide to align it like that?

Reply:
It's because it pushes the input fields down. That's why I removed the padding.

Reply:
I made some changes. I think it's much better now.